02-electric/electric-section-three/electric-section-three.css

```diff
@@ -1,65 +1,327 @@
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.electric-section-three-part-one-paragraph {
+  text-align: center;
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-one-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+/*****PART TWO******/
+
+.electric-section-three-part-two-container{
+  grid-area:electric-section-three-part-two-container;
+  display: grid;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  border: var(--subcontainer-border-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width:500px) {
+  .electric-section-three-part-two-container{
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.electric-section-three-part-two-image-container{
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-image-container {
+  }
+}
+
+.electric-section-three-part-two-image {
+  display: block;
+  width: 100%;
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-image {
+  }
+}
+
+.electric-section-three-part-two-copy-container{
+  display: grid;
+  align-content: start;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-copy-container {
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.electric-section-three-part-two-subtitle {
+  text-align: center;
+  font-size: var(--subtitle-font-size-mobile);
+  font-weight: var(--subtitle-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-subtitle {
+    text-align: left;
+    font-size: var(--subtitle-font-size-desktop);
+    font-weight: var(--subtitle-font-weight-desktop);
+  }
+}
+
+.electric-section-three-part-two-paragraph-container {
+  display: grid;
+  row-gap: var(--copy-container-row-gap-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-paragraph-container{
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.electric-section-three-part-two-paragraph {
+  text-align: center;
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 500px) {
+  .electric-section-three-part-two-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+/*****PART THREE******/
+
+.electric-section-three-part-three-container{
+  grid-area:electric-section-three-part-three-container;
+  display: grid;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  border: var(--subcontainer-border-mobile);
```

Comment:
looking at the figma
part two and part three do not have any borders on either mobile or desktop
but you added a border here ... is this deliberate?

Reply:
done